Adding a new category to the collections menu

raithos opened the pull request:
The thing I wanted to do in settings isn't going to work the way I want. I'm instead suggesting to move the checks to the collection database under a new category. I don't know if this will break anything for either builder, so I'd like you to have a look at it before pushing anything

raithos committed:
Adding a new category to the collections menu 

The thing I wanted to do in settings isn't going to work the way I want. I'm instead suggesting to move the checks to the collection database under a new category. I don't know if this will break anything for either builder, so I'd like you to have a look at it before pushing anything

geordanr commented on web.rb:
```diff
@@ -397,6 +398,7 @@ def initialize(user_id, expansions, singletons)
         self['user_id'] = user_id
         self['expansions'] = expansions
         self['singletons'] = singletons
+        self['checks'] = checks
```
`checks` isn't defined.  Like `expansions` and `singletons` it will need to be passed in as an argument to `Collections.new()` which is called in `get_collection()` (line 108) where you should set a sensible default for a brand new collection, as well as line 403 where you handle the case where you load a collection that may not already have a `checks` key (basically, all collections right now).  To match the existing behavior, add another `nil` added to match the new initializer signature, and on your end you should have some logic to deal with getting `null` back in that case.

raithos committed:
Lets just do some simple adjustments

I can be satisfied with a similar array as the other object arrays. Once inside I can work within it and setting it as a null value is fine since I'll be defining it on my side.

raithos reviewed (commented):
added null values
